brainstormforce/ultimate-addons-for-gutenberg
# Fix : HowTo - Showing blank page when deprecated file load.

### Description

- Created HowTo in V1.17.0 and switch to next-release it showing blank page.

### Screenshots

- https://share.getcloudapp.com/7KuPG7AJ

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a HowTo block in V1.17.0 and add times, cost, and step everything.
- Switch to `next-release` and check console and is page editable or not if not then
- Switch to `howto-deprecated-innerblocks-missing` and check no console issue and the page should be editable.

### Checklist:
